### ISSUE TITLE
Required `times` property for subscribe event

### ISSUE THREAD

**Issue by insertmike** [flekschas/regl-scatterplot#117]
Why is the `times` property required? Assuming it is used to specify the number of times the eventHandler function should be called when a specified event occurs, why enforcing this? 

Typescript complaining if not passed:

```ts
  scatter.subscribe<"select">("select", (event: EventMap["select"]) => {
     ...
  });
```

Error:

```
Expected 3 arguments, but got 2.ts(2554)
regl-scatterplot.esm.d.ts(73, 361): An argument for 'times' was not provided
```

In fact, where is this used, I can't find a reference for `times` in the repository, except at `PubSubEvent` definition. Is it retrieved somewhere dynamically from arg length?

https://github.com/flekschas/regl-scatterplot/blob/91aa3964e73aea532d63e10e958af453edbae3e7/src/types.d.ts#L210

**Comment by flekschas** [comment]
This is just a typing issue. The property is coming from https://github.com/flekschas/pub-sub and is not required. By default it's set to `Infinity`. I'll update the type in the next release, thanks for bringing this up.

**Comment by insertmike** [comment]
Fixed in ` v1.6.5` 💯
